src/ray/gcs/gcs_server/gcs_actor_manager.h

```diff
@@ -85,8 +85,6 @@ class GcsActor {
       // easy equivalent to class_name for.
       break;
     }
-
-    actor_table_data_.set_serialized_runtime_env(task_spec.serialized_runtime_env());
```

Comment:
@architkulkarni do you remember why we added this to actor table data? seems like we only ever need to get it from the task spec.

Reply:
I think it was to make it possible to display the runtime env info in the product, is that right @wuisawesome?

Reply:
Whoops just left the comment here: https://github.com/ray-project/ray/pull/18641/files#r710576046

But I think it's necessary because we have an optimization to clear the task spec early. (If we can find a way to get rid of that optimization, I'd be in favor of that though).

Reply:
Agreed, it seems really error-prone to just clear the task spec like this, but I think that should be a separate change